src/autoyast-rnc/firewall.rnc

```diff
@@ -24,7 +24,6 @@ y2_firewall =
   | FW_SERVICES_EXT_IP
   | start_firewall
   | enable_firewall
-  | default_zone
```

Comment:
Schubi, you added it also to the set of SuSEFirewall2 attributes, but it is firewalld specific.